source/views/streaming/streams/list.tsx

```diff
@@ -47,23 +64,53 @@ const useStreams = (date: Moment = moment.tz(timezone())) => {
 export const StreamListView = (): JSX.Element => {
 	let {data = [], error, reload, isPending, isInitial, isLoading} = useStreams()
 
+	let [filters, setFilters] = React.useState<ListType[]>([])
+
 	let entries = React.useMemo(() => {
-		return data
-			.filter((stream) => stream.category !== 'athletics')
-			.map((stream) => {
-				let date: Moment = moment(stream.starttime)
-				let dateGroup = date.format('dddd, MMMM Do')
-
-				let group = stream.status.toLowerCase() !== 'live' ? dateGroup : 'Live'
-
-				return {
-					...stream,
-					// force title-case on the stream types, to prevent not-actually-duplicate headings
-					category: titleCase(stream.category),
-					date: date,
-					$groupBy: group,
-				}
-			})
+		return data.map((stream) => {
+			let date: Moment = moment(stream.starttime)
+			let dateGroup = date.format('dddd, MMMM Do')
+
+			let group = stream.status.toLowerCase() !== 'live' ? dateGroup : 'Live'
+
+			return {
+				...stream,
+				// force title-case on the stream types, to prevent not-actually-duplicate headings
+				category: titleCase(stream.category),
+				date: date,
+				$groupBy: group,
+			}
+		})
```

Comment:
Could the body of the `map` be turned into a real function? It seems to be doing a fair bit of work.

Reply:
I'd be happy with that too.

Reply:
Sure, happy to extract this.